iphone/Classes/TiAlternateiconsModule.m

```diff
@@ -34,25 +34,7 @@ -(void)startup
 	// you *must* call the superclass
 	[super startup];
 
-	NSLog(@"[INFO] %@ loaded",self);
-}
-
--(void)shutdown:(id)sender
-{
-	// this method is called when the module is being unloaded
-	// typically this is during shutdown. make sure you don't do too
-	// much processing here or the app will be quit forceably
-
-	// you *must* call the superclass
-	[super shutdown:sender];
-}
-
-#pragma mark Cleanup
-
--(void)dealloc
-{
-	// release any resources that have been retained by the module
-	[super dealloc];
+	NSLog(@"[DEBUG] %@ loaded",self);
```

Comment:
Thanks for caching up this. I wasn't sure why the default template module from `appc new` was giving me a non-arc boilerplate.